pages/edit-company.vue

```diff
@@ -194,37 +194,61 @@
                   :rules="gofoundmeRule"
                 />
               </v-col>
-              <v-col cols="12" sm="4" xl="4">
```

Comment:
durch deine Änderungen haben die input felder auf voller größe die falsche breite. siehe screenshot -> 
<img width="774" alt="Bildschirmfoto 2020-04-12 um 11 52 05" src="https://user-images.githubusercontent.com/23628956/79065919-3007da80-7cb4-11ea-9cce-698cd64bdb9e.png">

Ich finde aus deinem neuen styling wird nicht ersichtlich das iban und owner zusammengehören. Daher hatte ich ja beides in einem row untereinander gesetzt.

Reply:
Ja da bin ich nicht so wirklich weitergekommen

Willst du das wieder umbauen?

Reply:
ja kann ich machen

Reply:
Prima!